src/complete_shell.rs

```diff
@@ -194,13 +194,13 @@ pub(crate) fn render_bash(
         match op {
             ShellComp::File { mask: None } => write!(res, "_filedir"),
             ShellComp::File { mask: Some(mask) } => {
-                writeln!(res, "_filedir '{}'", Shell(bashmask(mask)))
+                writeln!(res, "_filedir {}", Shell(bashmask(mask)))
             }
             ShellComp::Dir { mask: None } => write!(res, "_filedir -d"),
             ShellComp::Dir { mask: Some(mask) } => {
-                writeln!(res, "_filedir -d '{}'", Shell(bashmask(mask)))
+                writeln!(res, "_filedir -d {}", Shell(bashmask(mask)))
             }
-            ShellComp::Raw { bash, .. } => writeln!(res, "{}", Shell(bash)),
+            ShellComp::Raw { bash, .. } => writeln!(res, "{bash}"),
```

Comment:
I'm planning to bump MSRV at some point, but for now it's 1.56. The fix is trivial though

```suggestion
            ShellComp::Raw { bash, .. } => writeln!(res, "{}", bash),
```

Reply:
Didn't see the MSRV, apologies, and fixed!